### ISSUE TITLE
How to block or not the click event on expand or collapse correspondingly?

### ISSUE THREAD

**Issue by ahmadalibaloch** [Clans/FloatingActionButton#337]
I want it to block click events when the menu is open but do not block touch or click events when the menu is collapsed. 
I can't find a on Collapse or Expand Event. How to access this event so I can set setClickable to true or false when needed.

**Comment by vinzen** [comment]
+1

**Comment by jeanvallon** [comment]
+1

**Comment by jeanvallon** [comment]
Actually there is !

You need to use FloatingActionMenu in your layout (like it is shown in the readme.xml)
Then it is easy to collapse or expand your FAB menu.

```
FloatingActionsMenu fab = (FloatingActionsMenu) root.findViewById(R.id.multiple_actions);
        fab.collapse();
```

if this solves your problem, please close this issue.

**Comment by ahmadalibaloch** [comment]
No. It doesn't solve the problem. If you read the problem statement:

> I want it to block click events when the menu is open(expanded) but do not block touch or click events when the menu is collapsed.

**Comment by dudadayan** [comment]
+1

**Comment by wajdifadool** [comment]
check the new close(boolean animated) method ; 
`fabMenu.close(true)  `
this should close the issue
